prep-javascript-variables/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    var fullName = 'Cindy Le';
```

Comment:
make sure to declare all of the variables using `const` except for the never variable, which should be declared with `let`.

Reply:
I made the requested changes.